restapi/stories.go

```diff
@@ -67,13 +66,17 @@ func (api api) CreateStories(w http.ResponseWriter, r *http.Request) render.Rend
 	}
 
 	for i := 0; i < len(stories); i++ {
-		city := stories[i].CurrentCity
-		coordinates, err := api.locationFinder.GetCityCenter(city)
+		stories[i].CurrentCity = strings.Title(strings.ToLower(stories[i].CurrentCity))
+
+		// get number of stories with current city in db
+		var prevStories []models.Story
+		var numStoriesInCity int64
+		var err error
+		api.database.Where("current_city=?", stories[i].CurrentCity).Model(&prevStories).Count(&numStoriesInCity)
```

Comment:
Should we be also using https://gorm.io/docs/transactions.html#Transaction here?

Reply:
Hmm yeah using a transaction seems cleaner

Reply:
Used a dictionary since we cannot just increment one value as the city could be different on each iteration, and reading what you update in a transaction gives you the outdated value